{{ cookiecutter.package_name }}/pyproject.toml

```diff
@@ -87,3 +88,63 @@ write_to = "{{ cookiecutter.module_name }}/_version.py"
 {%- else -%}
 write_to = "{{ cookiecutter.module_name }}/version.py"
 {%- endif %}
+
+[tool.gilesbot]
+  [tool.gilesbot.pull_requests]
+  enabled = true
+
+  [tool.gilesbot.towncrier_changelog]
+  enabled = true
+  verify_pr_number = true
+  changelog_skip_label = "No Changelog Entry Needed"
+  help_url = "{{cookiecutter.sourcecode_url}}/blob/main/changelog/README.rst"
```

Comment:
We might want to template this based on github_repo instead as if this is a non-github URL then this wouldn't work. Not sure what we want to do if github_repo isn't set tho.